x-pack/plugins/ingest_manager/server/services/agents/checkin/state_new_actions.ts

```diff
@@ -134,10 +134,7 @@ export function agentCheckinStateNewActionsFactory() {
   const agentConfigs$ = new Map<string, Observable<FullAgentConfig | null>>();
   const newActions$ = createNewActionsSharedObservable();
   // Rx operators
-  const rateLimiter = createLimiter(
-    appContextService.getConfig()?.fleet.agentConfigRollupRateLimitIntervalMs || 5000,
-    appContextService.getConfig()?.fleet.agentConfigRollupRateLimitRequestPerInterval || 50
-  );
+  const conccurencyLimiter = createSubscriberConcurrencyLimiter();
```

Comment:
What do you think about `createSubscriberConcurrencyLimiter` accepting a number like `createLimiter` did?

i.e. so the change is 
`a(x, y)` -> `b(z)`
instead of 
`a(x, y)` -> `c = b(); c(z)`